mirgecom/initializers.py

```diff
@@ -425,8 +413,8 @@ def exact_rhs(self, discr, q, t=0.0):
         """
         Create the RHS for the lump-of-mass solution at time *t*, locations *x_vec*.
 
-        Note that this routine is only useful for testing under the condition of
-        uniform, and constant velocity field.
+        Note that *t* is used to advect the mass lumps under the assumption of
+        constant, and uniform velocity.
```

Comment:
(FYI) ReST also has a
```
.. note::

    Some text
```
environment that's useful for this type of thing.

Reply:
This really shouldn't be a *note* to self, but part of the documentation for the routine.

ee56b62